sci-mathematics/msieve/msieve-1.53.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="Library for Factoring Large Integers"
+DESCRIPTION="C library implementing a suite of algorithms to factor large integers"
+HOMEPAGE="https://sourceforge.net/projects/msieve/"
+COPYRIGHT="Public Domain"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/msieve/msieve/Msieve%20v$portVersion/msieve${portVersion/./}_src.tar.gz"
+CHECKSUM_SHA256="c5fcbaaff266a43aa8bca55239d5b087d3e3f138d1a95d75b776c04ce4d93bb4"
+PATCHES="msieve-1.53.patchset"
+
+ARCHITECTURES="?x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	msieve$secondaryArchSuffix = $portVersion
+	cmd:msieve$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libecm$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	msieve${secondaryArchSuffix}_devel = $portVersion
+	devel:libmsieve$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	"
```

Comment:
Oops, REQUIRES_devel is missing a dependency on the base package.
```diff
 REQUIRES_devel="
	haiku${secondaryArchSuffix}_devel
+	msive$secondaryArchSuffix == $portVersion base
 	"
```

Reply:
Why? It doesn't even use anything from the base, or is this a haikuports practice?

Reply:
Yes, you are right, but we allways add this dependency to make sure that:
1) if we uninstall the base package, then its devel package will also be uninstalled.
2) if we upgrade the base package to a new release, then it will force the devel package to also be upgraded.